qemu/tests/cfg/vioser_in_use.cfg

```diff
@@ -0,0 +1,54 @@
+- vioser_in_use:
+    type = vioser_in_use
+    suppress_exception = yes
+    file_transfer_serial_port = vs1
+    serials += " vs1"
+    serial_type_vs1 = virtserialport
+    transfer_timeout = 720
+    host_script = serial_host_send_receive.py
+    guest_script = VirtIoChannel_guest_send_receive.py
+    guest_scripts = VirtIoChannel_guest_send_receive.py;windows_support.py
+    driver_name = "vioser"
+    filesize = 100
+    clean_cmd = del /f /q
+    guest_script_folder = C:\
+    tmp_dir = %TEMP%
+    python_bin = python2.7
+    target_process = ${python_bin}.exe
+    Linux:
+        guest_script_folder = /var/tmp/
+        tmp_dir = /var/tmp/
+        clean_cmd = rm -f
+        filesize = 1000
+        python_bin = "`command -v python python3 | head -1`"
+        target_process = "python"
+    variants:
+        - guest_to_host:
+            file_sender = guest
+        - host_to_guest:
+            file_sender = host
+    variants:
+        - with_stop_continue:
+            only Windows
+            interrupt_test = subw_guest_pause_resume
+            suppress_exception = no
+            wait_timeout = 10
+        - with_shutdown:
+            only Windows
+            interrupt_test = shutdown_guest
+            shutdown_method = shell
+            shutdown_command = "shutdown -s -t 0"
+        - with_reboot:
+            only Windows
+            interrupt_test = reboot_guest
+            reboot_count = 1
+            reboot_method = shell
+        - with_system_reset:
+            only Windows
+            interrupt_test = reboot_guest
+            reboot_method = system_reset
+            sleep_before_reset = 0
```

Comment:
And what's this?

Reply:
Modified.
These two are not needed any more.
Thanks.